src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub struct Stealer<T: Send> {
 
 impl<T: Send> Clone for Stealer<T> {
     fn clone(&self) -> Self {
-        Stealer { deque: self.deque.clone() }
+        Stealer { deque: Arc::clone(&self.deque) }
```

Comment:
What's the reasoning given for this change?

Reply:
I remember reading this section from the docs: https://doc.rust-lang.org/std/sync/struct.Arc.html#cloning-references

> The Arc::clone(&from) syntax is the most idiomatic because it conveys more explicitly the meaning of the code. In the example above, this syntax makes it easier to see that this code is creating a new reference rather than copying the whole content of foo.